assets/src/edit-story/components/canvas/navLayer.js

```diff
@@ -32,7 +32,7 @@ import {
 
 function NavLayer() {
   return (
-    <Layer pointerEvents="none" onMouseDown={(evt) => evt.stopPropagation()}>
```

Comment:
@dvoytenko I think you added this, any idea what could go wrong here if we remove it?

Reply:
The things I'd check:
1. Starting lasso selection where we don't want (page carousel, page toolbar, etc)
2. Clearing out selection where we don't want